backend/de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerBankAccountImportHelper.java

```diff
@@ -79,7 +79,7 @@ else if (!Check.isEmpty(importRecord.getSwiftCode(), true) && !Check.isEmpty(imp
 			bankAccount = InterfaceWrapperHelper.newInstance(I_C_BP_BankAccount.class);
 			bankAccount.setC_BPartner_ID(bpartnerId.getRepoId());
 			bankAccount.setIBAN(importRecord.getIBAN());
-			bankAccount.setA_Name(importRecord.getSwiftCode());
+			bankAccount.setSwiftCode(importRecord.getSwiftCode());
```

Comment:
Put a name as well, please.
You can still use swift code as a name or generate a combination.